src/amo/components/LandingPage.js

```diff
@@ -108,9 +115,22 @@ export class LandingPageBase extends React.Component {
   }
 }
 
-export function mapStateToProps(state, ownProps) {
+export function mapStateToProps(
+  state, ownProps, _apiAddonType = apiAddonType, _log = log
+) {
+  let addonType;
+  try {
+    addonType = _apiAddonType(ownProps.params.visibleAddonType);
+  } catch (err) {
+    if (err.message.match('not found in API_ADDON_TYPES_MAPPING')) {
```

Comment:
Looking at es6-error I'm worried about the TODOs ("Better browser compatibility" and "Browser tests"). Given this site will likely be used on Firefox for iOS (right?) and I'd want it to be useful everywhere I don't want to use something that won't work there.

I'll try it as it's likely an easy patch but if it breaks in Safari or Edge that'd be annoying.